File: server/src/main/java/com/mbpolan/retrorealms/beans/requests/LoginRequest.java
package com.mbpolan.retrorealms.beans.requests;

/**
 * Message that contains a user's login request.
 *
 * @author mbpolan
 */
public class LoginRequest extends AbstractRequest {

    private String username;
    private String password;

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }
}

File: client/src/app/shared/messages/outgoing/login-request.ts
import {Message, MessageHeader} from "../message";

/**
 * Message that contains a request to login to the game.
 */
export class LoginRequest extends Message {

    private username: string;
    private password: string;

    public constructor(username: string, password: string) {
        super(MessageHeader.LOGIN);

        this.username = username;
        this.password = password;
    }
}